ca_description/urdf/create_base_gazebo.xacro

```diff
@@ -26,7 +26,7 @@
         <publishOdomTF>true</publishOdomTF>
         <publishWheelJointState>true</publishWheelJointState>
         <legacyMode>false</legacyMode>
-        <odometrySource>world</odometrySource>
+        <odometrySource>map</odometrySource>
```

Comment:
These changes are already in the original repo so you'll need to update your branches.